packages/expo/metro-config.js

```diff
@@ -1 +1 @@
-module.exports = require('@expo/metro-config');
\ No newline at end of file
+module.exports = require('@expo/cli/metro-config');
```

Comment:
Could we not just add a dependency to the expo package? I’m not especially in love with allowing imports into expo/cli:
1. This makes it harder to ncc in the future.
2. Allowing imports to expo cli is how we ended up with xdl.

Reply:
Yeah, we can also decide to move "ownership" to `expo` itself. Where ownership means "`expo` version dictates what versions of `@expo/metro-config` should be used for this SDK version".